aiida/orm/nodes/node.py

```diff
@@ -659,6 +657,18 @@ def is_valid_cache(self, valid: bool) -> None:
         'get_outgoing': 'get_outgoing',
     }
 
+    @classproperty
+    def Collection(cls):  # pylint: disable=invalid-name,no-self-use
```

Comment:
I'd note you could probably "re-route" this via `__getattr__` on the `AbstractNodeMeta` class (since it is a class level thing), but not crucial

Reply:
With `classproperty` this still functions properly on the class level right? Unless it is not working as it should, I'd leave this as is since it will be removed anyway.

Reply:
Yep no worries